cads_adaptors/tools/area_selector.py

```diff
@@ -78,30 +89,29 @@ def get_dim_slices(
     end: float,
     context: Context = Context(),
     longitude: bool = False,
+    precision: int = 2,
 ) -> list:
     da_coord = ds[dim_key]
 
     direction = bool(da_coord[0] < da_coord[1])  # True = ascending, False = descending
```

Comment:
Naming this variable `ascending` might make the if blocks more readable below (i.e. `if ascending` instead of `if direction`)
```suggestion
    ascending = bool(da_coord[0] < da_coord[1])  # True = ascending, False = descending
```